src/backend/distributed/citus.control

```diff
@@ -1,6 +1,6 @@
 # Citus extension
 comment = 'Citus distributed database'
-default_version = '10.0-3'
+default_version = '10.0-4'
```

Comment:
is this the only change we need to do? https://github.com/citusdata/citus/commit/db03afe91eabe333ba53a56f7d5d10c0ca13652b

Reply:
I believe it is, we are only increasing the patch version. The linked commit shows the changes for a minor version increase.